js/autocomplete-input.js

```diff
@@ -5,6 +5,15 @@ template.innerHTML = `<span style="display: inline-block; font-size 1em; positio
    <div id="suggestions" style="max-height:20rem; overflow-y:scroll; width: inherit; position: absolute; background-color:#fff;cursor:pointer !important; display: none; border: 1px solid gray; box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);"></div>
 </span>`
 
+/* TODO (Logan) move to an itemized input
```

Comment:
Has this been done yet? Is it a ticket?

Reply:
I don't think so but it may also not be needed I should double check and remove this if not